scripts/appbuilder

```diff
@@ -8,12 +8,10 @@
 function print_usage {
     echo "$(basename $0): a useful shell script for converting Docker images into Mystikos cpio archives"
     echo
-    echo "Usage: $(basename $0) [-o file] [-d [file]] [[-i] image] [-e extras] [-v]"
+    echo "Usage: $(basename $0) [-o file] [-i image] [-e extras] [-v] [[-d] file]"
     echo
     echo "  -d file     Build docker image from the named file"
-    echo "              Defaults to './Dockerfile' if no option specified"
```

Comment:
Why remove it?

Reply:
As we discussed, a docker file name is mandatory for default situation. 
```bash
appbulder <dockerfile-name>
```